src/app/components/manuscript-thumbnails-viewer/manuscript-thumbnails-viewer.component.ts

```diff
@@ -0,0 +1,110 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'evt-manuscript-thumbnails',
+  templateUrl: './manuscript-thumbnails-viewer.component.html',
+  styleUrls: ['./manuscript-thumbnails-viewer.component.scss'],
+})
+
+export class ManuscriptThumbnailsViewerComponent implements OnInit {
+  public grid = [];
+  public col = 2  ;
+  public row = 3;
+  public gridSize = this.col * this.row;
+  public indexPage = 0;
+  private items = [
```

Comment:
Add TODO comment or move this to a mockdata file.
This should probably be an `@Input()` of the conponent